signal_id/mask.py

```diff
@@ -232,10 +236,12 @@ def independent_cubes(self, struct=None):
     # -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
 
     def enable_backup(self):
-        self._implict_backup = True
+        self.is_backup_enabled = True
+        return self
 
     def disable_backup(self):
-        self._implict_backup = False
+        self.is_backup_enabled = False
+        return self
```

Comment:
You're right, this shouldn't be returning anything. I'm going to keep ```is_backup_enabled``` as is for now; there's a bunch of work to be done on the logging and it deserves its own PR.